www/review.php

```diff
@@ -11,6 +11,9 @@
 $state = SimpleSAML_Auth_State::loadState($stateId, ProfileReview::STAGE_SENT_TO_NAG);
 $logger = LoggerFactory::getAccordingToState($state);
 
+/* Skip the splash page for awhile to avoid annoying them with constant warnings. */
+ProfileReview::skipSplashPagesFor(24 * 60 * 60);
```

Comment:
I think this is one day but a variable would save future maintainers the pause of figuring that out, e.g, `$ONE_DAY = 24 * 60 * 60` or however you might do that in php.